sdk/storage/Azure.Storage.Common/src/Sas/SasQueryParameters.cs

```diff
@@ -169,6 +182,46 @@ public partial class SasQueryParameters
         /// </summary>
         public string ContentType => _contentType ?? string.Empty;
 
+
+        /// <summary>
+        /// Gets the Authorized AAD Object Id associated with the shared access signature.
+        /// The AAD Object ID of a user authorized by the owner of the User Delegation Key
+        /// to perform the action granted by the SAS. The Azure Storage service will
+        /// ensure that the owner of the user delegation key has the required permissions
+        /// before granting access but no additional permission check for the user specified
+        /// in this value will be performed. This cannot be used in conjuction with
+        /// <see cref="UnauthorizedAadObjectId"/>.
+        /// Only valid in an HNS enabled account. If this value is set in an non-HNS enabled
+        /// account, an authorization failure will be thrown.
+        /// </summary>
+        public string AuthorizedAadObjectId => _authorizedAadObjectId ?? string.Empty;
```

Comment:
Do we want these publically exposed as authorized and unauthorized?

Reply:
Maybe I will wait until we can come up with better names before I go and rename this.